experimental/sso-with-skills/javascript_nodejs/README.md

```diff
@@ -0,0 +1,88 @@
+
+## To try this sample
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
+
+- Create a bot registration in the azure portal for the `skillBot` and update [skillBot/.env](skillBot/.env) with the `MicrosoftAppId` and `MicrosoftAppPassword` of the new bot registration
+- Create a bot registration in the azure portal for the `rootBot` and update [rootBot/.env](rootBot/.env) with the `MicrosoftAppId` and `MicrosoftAppPassword` of the new bot registration
+- Update the `SkillAppId` variable in [rootBot/.env](rootBot/.env) with the `AppId` for the skill you created in the previous step
+- (Optionally) Add the `rootBot` `MicrosoftAppId` to the `AllowedCallers` comma separated list in [skillBot/.env](skillBot/.env)
+
+- Setup the 2 AAD applications for SSO as per steps given in [SkillBot AAD](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=srb%2Ccsharp#create-the-azure-ad-identity-application-1) and [RootBot AAD](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=sb%2Ccsharp#create-the-azure-ad-identity-application). You will end up with 2 aad applications - one for the skill consumer and one for the skill.
+- Create an aadv2 connection in the bot registration for the `SkillBot` and fill in values from the aadv2 application created for SSO, as per the [docs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=srb%2Ccsharp#create-azure-ad-connection-1). Update [SkillBot/appsettings.json](SkillBot/appsettings.json) with the `ConnectionName`  
+- Create an aadv2 connection in the bot registration for the `RootBot` and fill in values from the aadv2 application created for SSO, as per the [docs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=sb%2Ccsharp#create-azure-ad-connection). Update [RootBot/appsettings.json](SkillBot/appsettings.json) with the `ConnectionName`  
+
+- In a terminal, navigate to `samples\javascript_nodejs\82.sso-with-skills\skillBot`
+
+    ```bash
+    cd samples\javascript_nodejs\82.sso-with-skills\skillBot
+    ```
+
+- Install npm modules and start the bot
+
+    ```bash
+    npm install
+    npm start
+    ```
+
+- Open a **second** terminal window and navigate to `samples\javascript_nodejs\82.sso-with-skills\rootBot`
+
+    ```bash
+    cd samples\javascript_nodejs\82.sso-with-skills\rootBot
+    ```
+
+- Install npm modules and start the bot
+
+    ```bash
+    npm install
+    npm start
+    ```
+
+## Testing the bot using Bot Framework Emulator
+
+[Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
```

Comment:
We've (the docs team) have recently agreed to use "the" with the Emulator, but it's up to you, as long as you're consistent.
```suggestion
## Testing the bot using the Bot Framework Emulator

The [Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
```